src/sdk/client.ts

```diff
@@ -117,25 +118,31 @@ export const makeRequestFactory = (
         });
 
         return {
+          // TODO: Deprecate "data" field
```

Comment:
not sure if this is going to be what we end up doing

Reply:
Just chatted with ray and manas just now, lmk what you think